### ISSUE TITLE
This is empty... perhaps you should delete it or add the content from the paper to it

### ISSUE THREAD

**Issue by borgr** [grammatical/neural-naacl2018#1]

**Comment by EdwardZH** [comment]
I think it is hard to follow and too many details we do not know.

**Comment by borgr** [comment]
I see, so it is not expected to be published then? Perhaps you should write it in the documentation if the repo is not going to be deleted. 

What is then the best NMT-GEC that **is** open sourced? (best = high GLEU without training on non-public data I guess. If it is easy to change, even better, but can't expect everything, right?)
